lepton/package.go

```diff
@@ -26,6 +26,10 @@ func DownloadPackage(name string) string {
 }
 
 func ExtractPackage(archive string, dest string) {
+
+	// Remove the directory if it exits.
+	os.RemoveAll(dest)
```

Comment:
error is ignored since we don't have to fail if we can't remove the folder.

Reply:
This change removes not only package directory, but the whole .staging directory.
This means that boot.img, stage3.img and mkfs will be redownloaded each time.
In the case described at #193, it's not good, as I want to use fresh images - and this change leaves me no other choice than use downloaded images.